### DISCUSSION TITLE
New translation in Hungarian language.

### DISCUSSION THREAD

**Discussion by MGeri97** [sircharlo/meeting-media-manager#509] [Translations]
I would like to translate the program into Hungarian.

**Comment by sircharlo** [comment]
Thank you so much!

To get started, please [sign into GitLocalize](https://gitlocalize.com/) using your GitHub account, and then let me know when that's done by adding a comment in this chat. 

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by MGeri97** [reply]
I'm done. :)

**Reply by mtdvlpr** [reply]
You've been added :)

**Reply by mtdvlpr** [reply]
I also added you to the documentation gitLocalize project. That one is to translate the documentation website located at [sircharlo.github.io/meeting-media-manager/](https://sircharlo.github.io/meeting-media-manager/)

**Reply by sircharlo** [reply]
Let us know when you're done translating the app, we'll make sure that your work is included in the next release!
